tests/mocks/ERC721SafeMintableMock.cairo

```diff
@@ -1,12 +1,13 @@
+# SPDX-License-Identifier: MIT
```

Comment:
why doesn't it need the file name?

Reply:
I initially used the `openzeppelin-contracts` as a template and that repo doesn't include file names for mocks. Do you prefer we add them in?

Reply:
no need